pkgs/development/tools/jo/default.nix

```
@@ -1,15 +1,14 @@
 {stdenv, fetchFromGitHub, autoreconfHook}:
 
 stdenv.mkDerivation rec {
-  name = "jo-${version}";
-  version = "1.1";
+  name    = "jo-${version}";
+  version = "1.2";
 
   src = fetchFromGitHub {
-    owner = "jpmens";
-    repo = "jo";
-
-    rev = "v${version}";
-    sha256="1gn9fa37mfb85dfjznyfgciibf142kp0gisc2l2pnz0zrakbvvy3";
+    owner  = "jpmens";
+    repo   = "jo";
+    rev    = "${version}";
```

it's not necessary to interpolate `version`
```suggestion
    rev    = version;
```